index.d.ts

```diff
@@ -34,12 +35,20 @@ declare namespace NodePolyfillPlugin {
         | "vm"
         | "zlib"
 
-	export interface Options {
+	interface Include {
 		/**
-		By default, the modules that were polyfilled in Webpack 4 are mirrored over. However, if you don't want a module like console to be polyfilled you can specify alises to be skipped here.
-		*/
+                By default, the modules that were polyfilled in Webpack 4 are mirrored over. However, if you don't want a module like console to be polyfilled you can specify alises to be skipped here.
```

Comment:
Wrong way round?

Reply:
Thats what I get for doing this before I've had my coffee 😅